xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -231,7 +191,9 @@ bool CEdl::ReadEdl(const std::string& strMovie, const float fFramesPerSecond)
       }
       else if (strFields[i][0] == '#') // #12345 format for frame number
       {
-        iCutStartEnd[i] = (int64_t)(atol(strFields[i].substr(1).c_str()) / fFramesPerSecond * 1000); // frame number to ms
+        CLog::Log(LOGERROR, "%s - Frame number not supported in EDL files - supplied frame number: %s",
```

Comment:
this does not log the class name on Linux, which makes reading logs quite hard.

Reply:
Ok